### DISCUSSION TITLE
Why "--homooligomer 1" is used in the submit_example_6.sh of ProteinMPNN for homo-dimer and homo-trimer proteins?

### DISCUSSION THREAD

**Discussion by lanselibai** [RosettaCommons/rosetta#231] [General]
The submit_example_6.sh of ProteinMPNN designs homo-dimer and homo-trimer proteins
https://github.com/dauparas/ProteinMPNN/blob/main/examples/submit_example_6.sh

What is the meaning of `--homooligomer 1`?

Similarly, "--homo_oligomer 1" is used in the LigandMPNN. What is the meaning of "1" here?
https://github.com/dauparas/LigandMPNN?tab=readme-ov-file
```
python run.py \
        --model_type "ligand_mpnn" \
        --seed 111 \
        --pdb_path "./inputs/4GYT.pdb" \
        --out_folder "./outputs/homooligomer" \
        --homo_oligomer 1 \
        --number_of_batches 2
```

**Comment by roccomoretti** [comment]
Given that it's specific to ProteinMPNN/LigandMPNN instead of the Rosetta package _per se_, it's best to open such discussion questions on one of those repos.

That said, it's very common (in Rosetta as well) to represent true/false values as 1/0. Many programming languages and tools make parsing 0 & 1 into true and false easier than parsing strings like "true" and "false".

`--homo_oligomer 1` should be read like `--homo_oligomer true`  (though actually putting `true` may or may not work.)

**Reply by lanselibai** [reply]
Thank you very much! So I think, whatever the homooligomer is dimer, trimer, the software will consider all the chains as identical if `--homo_oligomer 1` is used.

Where are the dedicated repos for Protein/LigandMPNN?
